### PR TITLE
add predict_cls to predict_system

### PR DESCRIPTION
**Pull request by WenmuZhou**

### CODE REVIEW

File: tools/infer/predict_system.py [hunk 1]
```diff
@@ -88,6 +92,15 @@ def __call__(self, img):
             tmp_box = copy.deepcopy(dt_boxes[bno])
             img_crop = self.get_rotate_crop_image(ori_im, tmp_box)
             img_crop_list.append(img_crop)
+            cv2.imwrite(
+                '/home/zhoujun20/dygraph/PaddleOCR_rc/inference_results/{}.jpg'.
```
Comment (littletomatodonkey) [inline tools/infer/predict_system.py:96]: delete it?

File: tools/infer/predict_system.py [hunk 2]
```diff
@@ -88,6 +92,15 @@ def __call__(self, img):
...
+                format(bno), img_crop)
+        if self.use_angle_cls:
+            img_crop_list, angle_list, elapse = self.text_classifier(
+                img_crop_list)
+            print("cls num  : {}, elapse : {}".format(
```
Comment (littletomatodonkey) [inline tools/infer/predict_system.py:101]: print the tme unit at the sam time?

Comment (littletomatodonkey) [inline tools/infer/predict_system.py:101]: use logger info?

File: tools/infer/predict_system.py [hunk 3]
```diff
@@ -149,8 +159,13 @@ def main(args):
...
 
-            draw_img = draw_ocr(
-                image, boxes, txts, scores, drop_score=drop_score)
+            draw_img = draw_ocr_box_txt(
+                image,
+                boxes,
+                txts,
+                scores,
+                drop_score=drop_score,
+                font_path=font_path)
```
Comment (littletomatodonkey) [inline tools/infer/predict_system.py:168]: make it configurable?

File: tools/infer/predict_system.py (hunk 2 above)
Comment (WenmuZhou) [reply tools/infer/predict_system.py]: Only need to print the whole time is enough

**Pull request merged**